cts/update/array/positional-identifier.json

```diff
@@ -0,0 +1,71 @@
+{
+  "1-number": {
+    "request": {
+      "update": "composites",
+      "updates": [
+        {
+          "q": {
+            "_id": "array"
+          },
+          "u": {
+            "v.$[element]": {
+              "$numberInt": "100"
+            }
+          },
+          "arrayFilters": [
+            {
+              "element": {
+                "$numberInt": "42"
+              }
+            }
+          ]
+        }
+      ],
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "n": {
+        "$numberInt": "1"
+      },
+      "nModified": {
+        "$numberInt": "1"
+      },
+      "ok": {
+        "$numberDouble": "1.0"
+      }
+    }
+  },
+  "2-find": {
+    "request": {
+      "find": "composites",
+      "filter": {
+        "_id": "array"
+      },
+      "sort": {
+        "_id": {
+          "$numberInt": "1"
+        }
+      },
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "cursor": {
+        "firstBatch": [
+          {
+            "_id": "array",
+            "v.$[element]": {
+              "$numberInt": "100"
+            }
```

Comment:
Same as above, adding a field with key `v.$[element]` and value `100` doesn't quite make sense

Reply:
Fixed